cli_tools/osconfig_agent/osconfig.go

```diff
@@ -0,0 +1,26 @@
+//  Copyright 2018 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package main
+
+import (
+	osconfig "github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/osconfig_agent/_internal/osconfig/v1alpha1"
+)
+
+func lookupConfigs(svc *osconfig.Service, resource string) (*osconfig.LookupConfigsResponse, error) {
+	req := &osconfig.LookupConfigsRequest{
+		ConfigTypes: []string{"APT", "GOO", "YUM"},
```

Comment:
Can you use `packages.*Exists` here or is that coming in the future?

Reply:
This is just a placeholder for now for testing, will populate the types and osinfo based on the instance in the future,